Services/ChatHub.cs

```diff
@@ -27,6 +55,12 @@ public Task SendMessageToCallerAsync(string user, string message)
             return Clients.Caller.SendAsync("ReceiveMessageToCaller", user, message);
         }
 
+        public Task LastActiveRequest(int userId)
+        {
+            var user = _userService.GetUserById(userId);
+            return Clients.Caller.SendAsync("GetLastActive", user.LastActive);
```

Comment:
As far as I understand, this should also check if the user is currently active.

Reply:
Fixed!